packages/server-core/src/route/route/route.class.ts

```diff
@@ -23,18 +23,51 @@ All portions of the code written by the Ethereal Engine team are Copyright © 20
 Ethereal Engine. All Rights Reserved.
 */
 
-import type { Params } from '@feathersjs/feathers'
-import type { KnexAdapterParams } from '@feathersjs/knex'
-import { KnexService } from '@feathersjs/knex'
+import type { NullableId, Params } from '@feathersjs/feathers'
+import type { KnexAdapterOptions, KnexAdapterParams } from '@feathersjs/knex'
+import { KnexAdapter } from '@feathersjs/knex'
 
 import { RouteData, RoutePatch, RouteQuery, RouteType } from '@etherealengine/engine/src/schemas/route/route.schema'
+import { Id } from '@feathersjs/feathers'
+import { Application } from '../../../declarations'
 
 // eslint-disable-next-line @typescript-eslint/no-empty-interface
 export interface RouteParams extends KnexAdapterParams<RouteQuery> {}
 
-export class RouteService<T = RouteType, ServiceParams extends Params = RouteParams> extends KnexService<
+export class RouteService<T = RouteType, ServiceParams extends Params = RouteParams> extends KnexAdapter<
   RouteType,
   RouteData,
   RouteParams,
   RoutePatch
-> {}
+> {
+  app: Application
+
+  constructor(options: KnexAdapterOptions, app: Application) {
+    super(options)
+    this.app = app
+  }
+
+  async find(params?: RouteParams) {
+    if (params?.query?.paginate === false) {
```

Comment:
Should this be `typeof params?.query?.paginate !== undefined)` to catch if it's true or something else and delete it?

Reply:
I made it always remove the query param. There's still a check if it's false before setting params.paginate to it since params.paginate cannot be true, or anything other than false or some sort of explicit pagination param that I doubt we'll ever pass.